sklearn/cluster/optics_.py

```diff
@@ -146,6 +158,13 @@ class OPTICS(BaseEstimator, ClusterMixin):
         Point that a sample was reached from, indexed by object order.
         Seed points have a predecessor of -1.
 
+    clusters_ : array, shape (n_clusters, 2)
```

Comment:
The number of entries here would often be greater than `np.unique(optics.labels_)`, right? If so we need to be clearer.

Reply:
Perhaps this should be named `hierarchical_clusters_` or `cluster_hierarchy_`?

Reply:
done